### PR TITLE
Hide some part of the form if the person is inputing his/her own info…

### PR DESCRIPTION
**Pull request by Stephanie1125**
**The goal is to Hide some part of the form if the person is inputing his/her own information**

So inside the form user can choose 

+ I want to input information about this person
form

+ I am this person. (input my own information) **(Some not necessary part of the form is hidden)**
form2

also edit the place if user select 
+ This record is copied from another source
source

One remaining unsolved problem is that when you hide some part of the code,
it will appear a warning message to ask the user to fill all the required part (because the required part in the source section is hidden) :(

I have two idea to solve this problem:
1. define params.own_info to add {% if params.own_info %} to set the author_name automatically to the person's full_name.
2. edit the requred field to set the author name to not required. 

Do you have any better idea?

### CODE REVIEW

**Comment by Stephanie1125** [comment]
Please check it :)
Thank you very much

File: app/create.py [hunk 1]
```diff
@@ -58,11 +58,13 @@ def post(self):
         else:
             if not self.params.given_name:
                 return self.error(400, _('Name is required.  Please go back and try again.'))
-        if not self.params.author_name:
-            if self.params.clone:
-                return self.error(400, _('The Original author\'s name is required.  Please go back and try again.'))
-            else:
-                return self.error(400, _('Your name is required in the "Source" section.  Please go back and try again.'))
+        
+        if not self.params.own_info:
```
Comment (kevinalle) [inline app/create.py]: I think this will only check whether the own_info was provided. You should check whether it's set to yes or no

Comment (Stephanie1125) [inline app/create.py]: I see! 
Already edited it to ```if self.params.own_info == "no":```

Comment (kevinalle) [inline app/create.py]: As discussed in the meeting, you'll have to fill in author_name automatically if inputting own info.
So maybe a better option is to do something like this above:

if self.params.own_info == "yes":
  self.params.author_name = self.params.given_name

..etc

Comment (Stephanie1125) [inline app/create.py]: okay!

**Comment by Stephanie1125** [comment]
Please check it again :)
Thank you very much

**Comment by kevinalle** [comment]
Cool. LGTM

**Comment by Stephanie1125** [comment]
fixed #96

**Pull request merged**